tests/simtel/test_simtelfile.py

```diff
@@ -102,15 +102,15 @@ def test_show_event_is_not_empty_and_has_some_members_for_sure():
 
 def test_iterate_complete_file():
     expected_counter_values = {
+        prod4_path: 30,
         prod2_path: 8,
         prod3_path: 5,
-        prod4_path: 30,
     }
     for path, expected in expected_counter_values.items():
         try:
             for counter, event in enumerate(SimTelFile(path)):
                 pass
-        except (EOFError, IndexError):  # truncated files might raise these...
+        except EOFError:  # truncated files might raise these...
```

Comment:
grammar in comment "raise this" is better I think,.

Reply:
I think I need to add back the IndexError. We don't have a test file for this, but I think this is raised when the file is truncated inside a block parsed by our cython parsing methods that operate on byte arrays and try to index it beyond its bounds.

Reply:
ah ok.